yt/data_objects/region_expression.py

```diff
@@ -152,14 +152,17 @@ def _slice_to_edges(self, ax, val):
     def _create_region(self, bounds_tuple):
         left_edge = self.ds.domain_left_edge.copy()
         right_edge = self.ds.domain_right_edge.copy()
-        dims = []
+        dims = [1, 1, 1]
```

Comment:
note to reviewers: everything I do here, besides forcing dims to have 3 elements even in reduced dimensionality, isn't _necessary_, it's just small improvements I wrote while trying to fix the issue and I figured I might as well keep them.